Dynavity/Dynavity/view-model/canvas/CanvasViewModel.swift

```diff
@@ -142,7 +144,10 @@ class CanvasViewModel: ObservableObject {
 // MARK: Local storage autosaving
 extension CanvasViewModel {
     func saveCanvas() {
-        self.canvasRepo.save(model: self.canvas)
+        let annotationData = self.annotationCanvas.drawing.dataRepresentation()
```

Comment:
I think the `dataRepresentation()` part can be done when creating the canvas DTO